src/MCPClient/lib/clientScripts/policy_check.py

```diff
@@ -143,15 +164,15 @@ def _we_check_this_type_of_file(self):
         else:
             return True
 
-    def _file_is_derivative(self, for_access=False):
+    def _file_is_derivative(self, for_access: bool = False) -> bool:
         """Return ``True`` if the target file is a derivative; ``False``
         otherwise.
         """
         if self.is_manually_normalized_access_derivative:
             return True
         # Access derivatives have Derivation rows with NULL event types (cf.
         # normalize.py client script).
-        event_type = "normalization"
+        event_type: Optional[str] = "normalization"
         if for_access:
             event_type = None
```

Comment:
I think expressing this bit differently would allow the type checker to infer the possible values without the annotation:

```suggestion
        event_type = None if for_access else "normalization"
```